qutebrowser/completion/models/sortfilter.py

```diff
@@ -146,10 +148,44 @@ def filterAcceptsRow(self, row, parent):
                 data = self.srcmodel.data(idx)
                 if not data:
                     continue
-                elif self.pattern.casefold() in data.casefold():
-                    return True
+                else:
+                    matchers = {
+                        'contain': self.containMatcher,
+                        'fuzzy': self.fuzzyMatcher,
+                        'start': self.startMatcher
+                    }
+                    match_type = config.get('completion', 'match-type')
+                    if match_type not in matchers:
+                        match_type = 'contain'
```

Comment:
Why is this here? If the setting type makes sure it's `contain`/`fuzzy`/`start`, I don't see why this check would be needed.